lib/kbuildbarn/protoparse.go

```diff
@@ -10,17 +10,17 @@ const (
 	DefaultBBClientdScratchFileTemplate = "scratch/%s/%s"
 )
 
-type BBClientdList []*BBClientDLink
+type SymlinkList []*Symlink
```

Comment:
From the googling I'm doing, symbolic link is most commonly associated with soft link, and hard links are different. Can we name these Hardlink/HardlinkList (or even just Hardlinks)

Reply:
Done